### PR TITLE
Added PaddlePaddle classification model conversion case

### PR DESCRIPTION
**Pull request by jiangjiajun**
Provided dnn_model_runner module part for the classification model of PaddlePaddle and inference with OpenCV API

### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [x] I agree to contribute to the project under Apache 2 License.
- [x] To the best of my knowledge, the proposed patch is not based on a code under GPL or other license that is incompatible with OpenCV
- [x] The PR is proposed to proper branch
- [ ] There is reference to original bug report and related work
- [ ] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [x] The feature is well documented and sample code can be built with the project CMake

force_builders=linux,docs

### CODE REVIEW

File: samples/dnn/dnn_model_runner/dnn_conversion/requirements.txt [hunk 1]
```diff
@@ -7,3 +7,7 @@ torchvision>=0.6.1
 
 tensorflow>=2.1.0
 tensorflow-gpu>=2.1.0
+
+paddlepaddle>=2.0.0
+paddlepaddle-gpu>=2.0.0
+paddle2onnx>=0.5.1
```
Comment (asmorkalov) [inline samples/dnn/dnn_model_runner/dnn_conversion/requirements.txt:13]: Please extract the dependencies to separate files. Users that follow TF part of tutorial does not need it.

**Review by asmorkalov — CHANGES_REQUESTED** [review]

File: samples/dnn/dnn_model_runner/dnn_conversion/paddlepaddle/paddle_resnet50.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import paddle
+import paddlehub as hub
+import paddlehub.vision.transforms as T
+import cv2
```
Comment (vpisarev) [inline samples/dnn/dnn_model_runner/dnn_conversion/paddlepaddle/paddle_resnet50.py:4]: please, replace it with `import cv2 as cv` and then replace `cv2.` below with `cv.`

Comment (jiangjiajun) [reply samples/dnn/dnn_model_runner/dnn_conversion/paddlepaddle/paddle_resnet50.py]: Done

**Comment by vpisarev** [comment]
:+1:, @alalek, we can merge it, I think

**Review by vpisarev — APPROVED** [review]

**Comment by alalek** [comment]
Need to fix whitespace issues: http://pullrequest.opencv.org/buildbot/builders/precommit_docs/builds/30369

**Comment by jiangjiajun** [comment]
> Need to fix whitespace issues: http://pullrequest.opencv.org/buildbot/builders/precommit_docs/builds/30369

Problem solved.

**Pull request merged**